storage/aptosdb/src/fast_sync_aptos_db.rs

```diff
@@ -0,0 +1,221 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::AptosDB;
+use anyhow::{anyhow, Result};
+use aptos_config::config::{BootstrappingMode, NodeConfig};
+use aptos_crypto::HashValue;
+use aptos_storage_interface::{
+    cached_state_view::ShardedStateCache, state_delta::StateDelta, DbReader, DbWriter,
+    StateSnapshotReceiver,
+};
+use aptos_types::{
+    epoch_change::EpochChangeProof,
+    ledger_info::LedgerInfoWithSignatures,
+    state_store::{state_key::StateKey, state_value::StateValue, ShardedStateUpdates},
+    transaction::{TransactionOutputListWithProof, TransactionToCommit, Version},
+};
+use either::Either;
+use std::sync::{Arc, RwLock};
```

Comment:
nit: why not `use aptos_infallible::RwLock;`? You can get rid of the `expects` below.

Reply:
:trollface:  missed this one